sql/persistent_page_ids.sql

```diff
@@ -0,0 +1,7 @@
+
+CREATE TABLE IF NOT EXISTS /*_*/persistent_page_ids (
```

Comment:
`persistent_page_identifiers` felt a bit long and made it the longest table name.

Reply:
MediaWiki tables usually prefix with the initials of the extension name. So this would be `ppi_ids` or similar. Personally I don't care much for that convention and am also OK with the current name.